packaging/linux/deb/openra.lintian-overrides

```diff
@@ -0,0 +1,3 @@
+openra: duplicate-font-file usr/lib/openra/mods/common/FreeSans.ttf also in fonts-freefont-ttf
+openra: duplicate-font-file usr/lib/openra/mods/common/FreeSansBold.ttf also in fonts-freefont-ttf
+openra: binary-without-manpage usr/games/openra-server
```

Comment:
I guess you could symlink the font files, but that feels like a nastier hack than keeping everything self contained with a bit of duplication.